shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -403,16 +405,17 @@ private void setSystemChromeSystemUIOverlayStyle(
     // If transparent, SDK 29 and higher may apply a translucent scrim behind 2/3 button navigation
     // bars to ensure proper contrast. This can be overridden with
     // SystemChromeStyle.systemNavigationBarContrastEnforced.
-    if (systemChromeStyle.systemNavigationBarIconBrightness != null
-        && Build.VERSION.SDK_INT >= 26) {
-      switch (systemChromeStyle.systemNavigationBarIconBrightness) {
-        case DARK:
-          // View.SYSTEM_UI_FLAG_LIGHT_NAVIGATION_BAR
-          flags |= 0x10;
-          break;
-        case LIGHT:
-          flags &= ~0x10;
-          break;
+    if (Build.VERSION.SDK_INT >= 26) {
+      if (systemChromeStyle.systemNavigationBarIconBrightness != null) {
+        switch (systemChromeStyle.systemNavigationBarIconBrightness) {
+          case DARK:
+            // View.SYSTEM_UI_FLAG_LIGHT_NAVIGATION_BAR
```

Comment:
This one is also [deprecated](https://developer.android.com/reference/android/view/View#SYSTEM_UI_FLAG_LIGHT_NAVIGATION_BAR).